spec/models/user_spec.rb

```diff
@@ -71,4 +71,20 @@
       expect(user.has_already_queued?(video)).to_not be
     end
   end
+
+  describe '#generate_token!' do
+    it "updates the user's token field" do
+      user = Fabricate(:user)
+      expect(user.token).to_not be
+      user.generate_token!
+      expect(user.token).to be
```

Comment:
I would probably add a `reload` in this test because it is possible that the change isn't saved to the db but this spec would still pass because the object in memory was changed.